### PR TITLE
feat: update attachments api (Issue #497)

### PR DESCRIPTION
**Pull request by mikitabut**

### CODE REVIEW

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: failed

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: success

File: src/types/files.ts [hunk 1]
```diff
@@ -9,25 +9,29 @@ export type MIMEType =
   | ImageMIMEType
   | string;
 
-export interface BackendFile {
+export type BackendDataNodeType = 'ITEM' | 'FOLDER';
```
Comment (IlyaBondar) [inline src/types/files.ts]: I suggest to use string enum and use enum values on lines 23 and 28 and in `data-service.ts`

File: src/utils/app/data/data-service.ts [hunk 2]
```diff
@@ -262,8 +267,10 @@ export class DataService {
     bucket: string,
     parentPath?: string,
   ): Observable<FileFolderInterface[]> {
+    const filter: BackendDataNodeType = BackendDataNodeType.FOLDER;
```
Comment (IlyaBondar) [inline src/utils/app/data/data-service.ts]:
```suggestion
    const filter = BackendDataNodeType.FOLDER;
```

File: src/utils/app/data/data-service.ts [hunk 3]
```diff
@@ -303,8 +310,10 @@ export class DataService {
     bucket: string,
     parentPath?: string,
   ): Observable<DialFile[]> {
+    const filter: BackendDataNodeType = BackendDataNodeType.ITEM;
```
Comment (IlyaBondar) [inline src/utils/app/data/data-service.ts]:
```suggestion
    const filter = BackendDataNodeType.ITEM;
```

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**